mitreattack/collections/collection_to_index.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import os
+from tqdm import tqdm
+import json
+import uuid
+from dateutil.parser import isoparse
+import re
+
+
+class CollectionToIndex:
+    @staticmethod
+    def generate_index(name, description, root_url, files=None, folders=None, bundles=None):
+        """generate a collection index from the input data and return the index as a dict
+        arguments:
+            name (string): the name of the index
+            description (string): the description of the index
+            root_url (string): the root URL where the collections can be found. Specified collection paths will be
+                                appended to this for the collection URL
+            files (string[], optional): collection JSON files to include in the index. Cannot be used with folder
+                                argument
+            folders (string[], optional): folders of collection JSON files to include in the index. Cannot be used with
+                                files argument. Will only match collections that end with a version number
+            bundles (stix bundle[], optional): array of stix bundle objects to include in the index
+            output (string, optional): filename for the generated collection index file
```

Comment:
I don't believe the output argument is actually used (or even present). [This seems to be an issue with the original script as well.](https://github.com/mitre-attack/attack-stix-data/blob/master/util/generate-collection-index.py#L17)

Reply:
Huh, that's definitely an intriguing discovery .-. I'll remove the extraneous documentation (never thought I'd say that one).